library/src/webapp/js/sakai-table-toolbar/searchFilterPanelMacro.js

```
@@ -1,44 +1,54 @@
+        field._ac = new AbortController();
+        try {
+            const tmp = document.createElement('div');
+            tmp.innerHTML = await (await fetch(url, { credentials: 'same-origin', signal: field._ac.signal })).text();
+
+            const newTargets = tmp.querySelectorAll(sel);
+            targets.forEach((el, i) => { if (newTargets[i]) el.innerHTML = newTargets[i].innerHTML; });
+
+            history.replaceState(null, '', url);
+            document.dispatchEvent(new CustomEvent('sfp:updated'));
+        } catch (e) {
+            if (e.name !== 'AbortError') location = url;
```

Biome (2.4.4)

[error] A global variable should not be reassigned.

(lint/suspicious/noGlobalAssign)